### PR TITLE
create navbar

### PR DESCRIPTION
**Pull request by KetiRudneva**

### CODE REVIEW

**Comment by VKhayretdinov** [comment]
Unfortunately, I don't see the task number and the task link, so I can't figure out what the task is related to this PR.
I'd like you to review the PR creation guide (it's located in Slack)

**Comment by KetiRudneva** [comment]
Here there is the link on task: https://aspirity.atlassian.net/browse/PR2022-80

File: package.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
   "dependencies": {
     "@emotion/react": "11.7.1",
     "@emotion/styled": "11.6.0",
+    "@material-ui/core": "^4.12.3",
```
Comment (NKonstantin007) [inline package.json:14]: Do not use the MUI v4. Use only v5. So delete this package and update imports

**Review by NKonstantin007 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "@emotion/react": "11.7.1",
     "@emotion/styled": "11.6.0",
     "@mui/icons-material": "5.3.1",
-    "@mui/material": "5.3.1",
+    "@mui/material": "^5.4.1",
+    "@mui/styles": "^5.4.1",
```
Comment (NKonstantin007) [inline package.json:16]: ATTENTION!!! Never never never use the `^` symbol in this project in package versions. It's the reason of conflicts in yarn.lock file which bright the big problems for us lately

File: src/modules/Auth/components/ButtonMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
```
Comment (NKonstantin007) [inline src/modules/Auth/components/ButtonMenu.tsx:31]: You can pass this styles as a properties:
```suggestion
      <Box dispaly="flex" alignItems="center" textAlign="center">
```
I prefer to use properties instead `sx`. But your solution works too and you don't have to change it according to my proposal

File: src/modules/Auth/components/ButtonMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        id="account-menu"
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        PaperProps={{
+          elevation: 0,
+          sx: {
+            overflow: 'visible',
+            filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
+            mt: 4.5,
```
Comment (NKonstantin007) [inline src/modules/Auth/components/ButtonMenu.tsx:56]: As you can see the spacing coefficient in the theme is 4 px. It means that if you set the value of css properties which can be measured in pixels this value multiple to this coefficient. So if you set 2 the value is 8px, 3 - 12 px and etc.
Of course you can set the float value but I advise to use only integer values if you want mui transform it to pixel by multiplying to the coefficient. If you need the value which not multiple of 4 I would set the explicit value with units In this case it '18px'.
image

Comment (NKonstantin007) [reply src/modules/Auth/components/ButtonMenu.tsx:56]: And can you check all places where you pass the float numbers and update its to explicitly values with units

File: src/modules/Auth/components/ButtonMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import Logout from '@mui/icons-material/Logout';
+import Settings from '@mui/icons-material/Settings';
```
Comment (NKonstantin007) [inline src/modules/Auth/components/ButtonMenu.tsx:3]: Add suffix Icon in the end for all icons
```suggestion
import SettingsIcon from '@mui/icons-material/Settings';
```

File: src/modules/Auth/components/ButtonMenu.tsx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+      >
+        <MenuItem>
+          <ListItemIcon>
+            <Settings fontSize="small" />
+          </ListItemIcon>
+          Change your photo
+        </MenuItem>
+        <MenuItem className={classes.logOut}>
+          <ListItemIcon>
+            <Logout fontSize="small" className={classes.logOut} />
```
Comment (NKonstantin007) [inline src/modules/Auth/components/ButtonMenu.tsx:88]: If you pass color="#FF0000" it won;t work?
```suggestion
            <Logout fontSize="small" color="#FF0000" />
```

File: src/modules/Auth/components/Navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  },
+  title: {
+    flexGrow: 1,
+  },
+  avatar: {
+    marginRight: '20px',
+  },
+}));
+
+export const Navbar = () => {
```
Comment (NKonstantin007) [inline src/modules/Auth/components/Navbar.tsx:23]: I would rename this component to TopBar

File: src/modules/Auth/components/Navbar.tsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+  return (
+    <AppBar position="absolute" className={classes.appBar}>
+      <Toolbar>
+        <Typography variant="h6" className={classes.title}>
+          LOGO
+        </Typography>
+        <Avatar
+          alt="Cindy Baker"
+          src="/static/images/avatar/3.jpg"
```
Comment (NKonstantin007) [inline src/modules/Auth/components/Navbar.tsx:34]: I don't found this image. Is it just example?

File: src/modules/Auth/components/ButtonMenu.tsx (hunk 5 above)
Comment (KetiRudneva) [reply src/modules/Auth/components/ButtonMenu.tsx:3]: eslint prefer this order :D

File: src/modules/Auth/components/Navbar.tsx (hunk 8 above)
Comment (KetiRudneva) [reply src/modules/Auth/components/Navbar.tsx:24]: Yes, it's an example

File: src/modules/Auth/components/ButtonMenu.tsx (hunk 3 above)
Comment (KetiRudneva) [reply src/modules/Auth/components/ButtonMenu.tsx:23]: I leave it like this

**Review by NKonstantin007 — APPROVED** [review]

**Pull request merged**